src/vstart.sh

```diff
@@ -1017,15 +1017,9 @@ $BLUESTORE_OPTS
         ; kstore
         kstore fsck on mount = true
 EOF
-    if [ "$crimson" -eq 1 ]; then
-        wconf <<EOF
-        crimson osd objectstore = $objectstore
-EOF
-    else
-        wconf <<EOF
+    wconf <<EOF
```

Comment:
This line is shouldn't be here, we should revert same as: https://github.com/ceph/ceph/pull/60821/commits/15e746bf5b1b4f12d7bc2d8b0ac0b4baf3043036

```
        kstore fsck on mount = true
        osd objectstore = $objectstore
EOF
```